swift/ios-shoppe-demo/Resources/Info.plist

```diff
@@ -2,6 +2,10 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>FSOrgId</key>
+	<string>KWH</string>
+	<key>FSHost</key>
+	<string>staging.fullstory.com</string>
```

Comment:
Let's use production since this is what external devs would default to us.  (You could also swap locally.)